docs/admin.md

```diff
@@ -1,17 +1,22 @@
 # Admin
 Welcome to the Admin documentation this page explains some of the configuration options for news.
 ## System Cron
-Nextcloud uses Cron to run regular jobs, News relies on the Job system to execute the feed updates.
+Nextcloud uses cron to run regular jobs, News relies on the Job system to execute the feed updates.
 Alternatively you may use an [external updater](https://nextcloud.github.io/news/clients/#update-clients), in this case you need to disable the system cron in the settings.
 
 ## Auto purge count
-This value represents the maximum amount of read items per feed, which won't be deleted by the cleanup job.
-For example if the value is 200 there can be maximum 200 read items per feed, unread items are unaffected.
+This value represents the maximum amount of read items (articles) per feed, which won't be deleted by the cleanup job.
+For example, if the value is 200 there can be maximum 200 read items per feed, unread items are unaffected.
 If old articles reappear after being read, try to increase this value.
```

Comment:
This shouldn't happen, since at least `articles_per_update` items are always kept, no matter what threshold is set.
https://github.com/nextcloud/news/blob/6f9d3bdbcd748f1b7a73d51b2acc23b88c001301/lib/Db/ItemMapperV2.php#L227

However, I also have a feed that occasionally refetches the oldest item, which is why I am still looking for why the `articles_per_update` value is sometimes 1 too low.

Reply:
Yea I'm interested in getting a clear picture, this is how it was always documented I just added the part about the unread items due to the new option. Which I'm also not a 100% sure if it will work exactly like that.